src/main/java/duke/rahbot/Duke.java

```diff
@@ -97,6 +111,11 @@ public void start(Stage stage) {
         stage.show();
     }
 
+    /**
+     * Adds event handlers for user input, including button clicks and text field submissions.
+     * This method ensures that when the user interacts with the input controls, the appropriate
+     * actions are taken to process and respond to the input.
+     */
     private void addInputHandlers() {

```

Comment:
I like how you used camelCase and action verb at the start of method names, which follows coding standards! 👍